### DISCUSSION TITLE
Youtube RSS

### DISCUSSION THREAD

**Discussion by akasaka99** [ViennaRSS/vienna-rss#1442] [General]
Hello-

I'm having an issue subscribing to Youtube RSS feeds for some type of Youtube channels specifically the ones with channel names.

I know that for the channels which are of this format:

1. If channel_url = https://www.youtube.com/channel/CHANNEL_ID
Then RSS feed = https://www.youtube.com/feeds/videos.xml?channel_id=CHANNEL_ID

2. If channel_page =  https://www.youtube.com/user/USER_HANDLE
Then RSS feed = https://www.youtube.com/feeds/videos.xml?user=USER_HANDLE
 
However, for channels which urls have the following format neither of the above works:
https://www.youtube.com/c/CHANNEL_NAME like for instance https://www.youtube.com/c/censiCLICK
I cant subscribe to it.

Many thanks for help !

**Comment by TAKeanice** [comment]
Try the new browser as soon as we release it! I just tested and can confirm it worked to subscribe to the channel. The problem is probably in the way that the URLs for feeds are searched currently, in combination with youtube only loosely adhering to standards (rss link should be in the header)

**Reply by akasaka99** [reply]
hello,
I'm using Mac's "Vienna 3.7.1 :d356d926: is currently the newest version available."
So as a matter of example when the new release becomes available in the case of  https://www.youtube.com/c/censiCLICK how should the RSS sub look like?
The only type of youtube url where i can still subscribe are of the following type
1) https://www.youtube.com/channel/CHANNEL_ID
2)  https://www.youtube.com/user/USER_HANDLE
None of those two urls correspond to the url type https://www.youtube.com/c/CHANNEL_ID_Name
Many thanks

**Reply by TAKeanice** [reply]
For `youtube.com/c/CHANNEL_NAME` (which really exists) the feed is `youtube.com/feeds/videos.xml?channel_id=UCminsc7LLZLAhiPcbgl7zbg`
The new browser will detect those links on the channel start page. If you are want to find them right away, you could visit the channel in Safari, inspect the page sourcecode and search for videos.xml

**Reply by TAKeanice** [reply]
Please excuse typos in the feed URL, since I manually copied it using my mobile phone.

**Reply by akasaka99** [reply]
No worries! Thank you for your help!
I will wait for the new version and will come back if I have any issue.
Best
